### PR TITLE
Removal of RHM notice

### PR DESCRIPTION
**Pull request by JensTrier**
Remove the RHM notice from phone number assignment cmdlets.

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsUser.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Modifies Skype for Business properties for an existing user account.
 Properties can be modified only for accounts that have been enabled for use with Skype for Business.
 This cmdlet was introduced in Lync Server 2010.
 
-**Note**: Using this cmdlet for Microsoft Teams users will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) and [Remove-CsPhoneNumberAssignment](/powershell/module/teams/remove-csphonenumberassignment) cmdlets instead. The new cmdlets are currently not supported for customers and tenants that are or have been enabled for Regionally Hosted Meetings for Skype for Business Online. These customers should continue to use Set-CsUser.
+**Note**: Using this cmdlet for Microsoft Teams users will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) and [Remove-CsPhoneNumberAssignment](/powershell/module/teams/remove-csphonenumberassignment) cmdlets instead.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:20]:
```suggestion
**Note**: Using this cmdlet will be deprecated for Microsoft Teams users. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) and [Remove-CsPhoneNumberAssignment](/powershell/module/teams/remove-csphonenumberassignment) cmdlets instead.
```

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 2]
```diff
@@ -39,14 +36,11 @@ If the cmdlet executes successfully, no result object will be returned. If the c
 and a Message string parameter with additional details of the failure.
 
 **Note**: In Teams PowerShell Module 4.2.1-preview and later we are changing how the cmdlet reports errors. Instead of using a result object, we will be generating an
-
-exception in case of an error and we will be appending the exception to the $Error automatic variable. The cmdlet will also
-now support the -ErrorAction parameter to control the execution after an error has occured.
+exception in case of an error and we will be appending the exception to the $Error automatic variable. The cmdlet will also now support the -ErrorAction parameter to
+control the execution after an error has occured.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:40]:
```suggestion
control the execution after an error has occurred.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested change. Many thanks.

File: skype/skype-ps/skype/Set-CsUser.md (hunk 1 above)
Comment (JensTrier) [reply skype/skype-ps/skype/Set-CsUser.md:20]: @JohanFreelancer9 It is only for Microsoft Teams user that the cmdlets will be deprecated. The PS page is also used for Skype for Business Server

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @get-itips @tseward

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JensTrier** [comment]
Made only a small change to remove-csphonenumberassignment to force re-validation of the PR

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**